src/main/java/de/numcodex/feasibility_gui_backend/api/QueryHandlerRestController.java

```diff
@@ -54,6 +58,16 @@ public Response runQuery(
             .pathSegment("api", "v1", "query-handler", "result", id)
             .build()
             .toUri();
+
+    if(apiBaseUrl.length() > 0){
```

Comment:
The value parameter within the constructor does not have a default value. Thus, this will throw a `NullPointerException` if `app.apiBaseUrl` is simply not defined.

Reply:
fixex with your suggested change below